vcd/sample_vcd_test_config.json

```diff
@@ -11,7 +11,14 @@
     "allowInsecure": true,
     "//": "tfAcceptanceTests is needed to allow the whole test suite to run",
     "//": "Warning: it may cost time (from 14 to 40 min) and possibly money",
-    "tfAcceptanceTests": false
+    "tfAcceptanceTests": false,
```

Comment:
Why don't we set tfAcceptanceTests to true in the sample by default? Seems an extra hassle to change this and easy to miss. Actually, do we need to change this to true of you run `make testacc`?

Reply:
fixed